### DISCUSSION TITLE
Need to recheck some of the template id

### DISCUSSION THREAD

**Discussion by daffainfo** [projectdiscovery/nuclei-templates#11276] [Q&A] [answered]
https://github.com/search?q=repo%3Aprojectdiscovery%2Fnuclei-templates+%22id%3A+smb-signing-not-required%22&type=code

For example, there are 2 different templates that uses the same id

**Comment by GeorginaReeder** [comment]
Hi there! Thanks for bringing this up :) tagging @dwisiswant0 to take a look.

**Comment by dwisiswant0** [comment]
i believe it would make sense to deprecate `code/windows/audit/smb-signing-not-required.yaml` since the other templates are platform-agnostic & implemented using JavaScript protocol, which provides consistency across different environments.

cc: @princechaddha & @pussycat0x

**Reply by princechaddha** [reply, marked as answer]
Yes, but each functions differently, one is used to check smb misconfig on other targets, while the Windows one is designed for local checks on the host

**Comment by princechaddha** [comment]
Thank you @daffainfo, I have updated the names and it should be correct now
